test/integration/parseEntries.test.ts

```diff
@@ -0,0 +1,498 @@
+          },
+          id: '4SEhTg8sYJ1H3wDAinzhTp',
+          type: 'Entry',
+          createdAt: '2021-04-15T12:27:36.723Z',
+          updatedAt: '2021-04-15T12:27:36.723Z',
+          environment: {
+            sys: { id: 'master', type: 'Link', linkType: 'Environment' },
+          },
+          revision: 1,
+          contentType: { sys: { type: 'Link', linkType: 'ContentType', id: 'cat' } },
+          locale: 'en-US',
+        },
+        fields: {
+          name: 'CatWithUnpublishedFriend',
+          likes: ['dogs'],
+          color: 'black',
+          bestFriend: {
+            sys: {
+              type: 'Link',
+              linkType: 'Entry',
+              id: '6SiPbntBPYYjnVHmipxJBF',
+            },
+          },
+          birthday: '2020-07-02T00:00:00Z',
+          lives: 9,
+          image: { sys: { type: 'Link', linkType: 'Asset', id: 'happycat' } },
+        },
+      },
+    ],
+    errors: [
+      {
+        sys: { id: 'notResolvable', type: 'error' },
+        details: { type: 'Link', linkType: 'Entry', id: '6SiPbntBPYYjnVHmipxJBF' },
+      },
+    ],
+    includes: {
+      Asset: [resolvedHappyCatAsset],
+    },
+  }
+  dataWithUnresolvableLinkAndAllLocales = {
+    total: 1,
+    skip: 0,
+    limit: 100,
+    items: [
+      {
+        metadata: { tags: [] },
+        sys: {
+          space: {
+            sys: { type: 'Link', linkType: 'Space', id: 'ezs1swce23xe' },
+          },
+          id: '4SEhTg8sYJ1H3wDAinzhTp',
+          type: 'Entry',
+          createdAt: '2021-04-15T12:27:36.723Z',
+          updatedAt: '2021-04-15T12:27:36.723Z',
+          environment: {
+            sys: { id: 'master', type: 'Link', linkType: 'Environment' },
+          },
+          revision: 1,
+          contentType: { sys: { type: 'Link', linkType: 'ContentType', id: 'cat' } },
+          locale: 'en-US',
+        },
+        fields: {
+          name: {
+            'en-US': 'CatWithUnpublishedFriend',
+            tlh: 'CatWithUnpublishedFriend',
+          },
+          likes: {
+            'en-US': ['dogs'],
+          },
+          color: {
+            'en-US': 'black',
+          },
+          bestFriend: {
+            'en-US': {
+              sys: {
+                type: 'Link',
+                linkType: 'Entry',
+                id: '6SiPbntBPYYjnVHmipxJBF',
+              },
+            },
+          },
+          birthday: {
+            'en-US': '2020-07-02T00:00:00Z',
+          },
+          lives: {
+            'en-US': 9,
+          },
+          image: {
+            'en-US': { sys: { type: 'Link', linkType: 'Asset', id: 'happycat' } },
+          },
+        },
+      },
+    ],
+    errors: [
+      {
+        sys: { id: 'notResolvable', type: 'error' },
+        details: { type: 'Link', linkType: 'Entry', id: '6SiPbntBPYYjnVHmipxJBF' },
+      },
+    ],
+    includes: {
+      Asset: [resolvedHappyCatAssetAllLocales],
+    },
+  }
+})
+
+// expand to cover also localeClient
```

Comment:
This is already done, right?

Reply:
We're not using the `localeClient` but we have locales in the input data, so removed it.